scripts/validate_docstrings.py

```diff
@@ -725,7 +740,21 @@ def get_validation_data(doc):
             if not doc.parameter_desc(param)[0].isupper():
                 errs.append(error('PR08', param_name=param))
             if doc.parameter_desc(param)[-1] != '.':
```

Comment:
Instead of defining a local function within this branch can you update the condition instead?

Reply:
I have also thought about that, but I was not sure how to translate the recursive function into a condition.

Do you think it is better if I make the function into a method of the class Docstring?
For example:
```
        def parameter_end_with_bullet(self, param):
        # Return True if a parameter description ends
        # with a bullet point
        for desc_line in self.parameter_desc_list(param)[::-1]:
            if desc_line[0] in ['*', '-', '+']:
                return True
            elif desc_line[0:2] != '  ':
                return False
            else:
                pass
```

Reply:
Maybe a classmethod called `properly_terminated_description` (kind of verbose) or something similar

Reply:
Hey @WillAyd ,
I have add the method and amended the condition.
Is it what you had in mind?